docs/guide/overview.asciidoc

```diff
@@ -38,6 +36,69 @@ This data is buffered for a short period and sent on to APM Server.
 It listens on port 8200 by default and receives data from agents through a JSON HTTP API.
 Then it creates documents from that data and stores them in Elasticsearch.
 
-To visualize the data after it's sent to Elasticsearch,
-you can use the the dedicated APM UI bundled in X-Pack,
-or the pre-built open source Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
\ No newline at end of file
+*Elasticsearch* is a highly scalable open-source full-text search and analytics engine.
+It allows you to store, search, and analyze big volumes of data quickly and in near real time.
+Elasticsearch is used to store APM performance metrics and make use of its aggregations. 
+
+*Kibana* is an open source analytics and visualization platform designed to work with Elasticsearch.
+You use Kibana to search, view, and interact with data stored in Elasticsearch.
+You an use Kibana to visualize APM data by utilizing the dedicated APM UI bundled in Basic license,
+or the pre-built, open source,
+Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
+
+[[concepts]]
+[float]
+=== APM Concepts
+
+[float]
+==== Events
+APM agents capture different types of information from within their instrumented applications, known as events.
+Events can be <<errors,`Errors`>>, <<transaction-spans,`Spans`>>, or <<transactions,`Transactions`>>.
+These events are then streamed to a single endpoint in the APM Server which sorts and processes the events. 
+
+*<<errors,`Errors`>>* contain information about the original exception that occurred,
+or about a log created when the exception occurred.
+
+*<<transaction-spans,`Spans`>>* contain information about a specific code path that has been executed.
+They measure from the start to end of _something_,
+and they can have a parent/child relationship with other spans. 
+
+*<<transactions,`Transactions`>>* are a special kind of span that have metadata associated with them.
+You can think of transactions as the highest level of work you're measuring within a service.
+E.g., serving an HTTP request or running a specific background job.
+
+View the APM <<apm-data-model>> documentation for more information on Events. 
```

Comment:
Wouldn't just using an attribute like this simply render a naked link within the text? I'm thinking it might be prettier to make part of the sentence into a link instead similar to how we do in the rest of the document - but it's up to you of course - I just wanted to mention it in case it slipped passed your radar 😃

Reply:
No actually. When you link like this it takes the section title and inserts it into the link text. So `<<apm-data-model>>` renders as:
<img width="579" alt="screen shot 2018-12-06 at 11 01 53 am" src="https://user-images.githubusercontent.com/5618806/49605815-73c33500-f946-11e8-8653-c070da972651.png">

Reply:
Ah, nice - didn't know that 👍 Carry on